gcalcli/gcal.py

```diff
@@ -633,7 +633,7 @@ def _tsv(self, start_datetime, event_list):
 
             if self.details.get('email'):
                 output += '\t%s' % (event['creator']['email'].strip()
-                                    if 'email' in event['creator'] else '')
+                                    if 'creator' in event and 'email' in event['creator'] else '')
```

Comment:
Thanks for finding this!

Can we solve it with a `.get` instead of a conjunction?

```
if 'email' in event.get('creator', {}) else ''
```

Reply:
Updated